Okta.AspNet/JwtOptionsBuilder.cs

```diff
@@ -24,9 +24,10 @@ public class JwtOptionsBuilder
         /// <summary>
         /// Builds the JwtBearerAuthenticationOptions object used during the authentication process.
         /// </summary>
+        /// <param name="authenticationType">The authentication type.</param>
         /// <param name="options">The Okta options.</param>
         /// <returns>An instance of JwtBearerAuthenticationOptions.</returns>
-        public static JwtBearerAuthenticationOptions BuildJwtBearerAuthenticationOptions(OktaWebApiOptions options)
+        public static JwtBearerAuthenticationOptions BuildJwtBearerAuthenticationOptions(string authenticationType, OktaWebApiOptions options)
```

Comment:
Since this is a public method, it would probably be preferable to maintain the old signature and just add a new overload that takes `authenticationType`.  Just in case someone used this in it's old form, they won't have to rewrite every invocation.  You could just re-order the parameters so that `authenticationType` is last in the list and make it optional by giving it a default value, for example: `BuildJwtBearerAuthenticationOptions(OkeaWebApiOptions options, string authenticationType = OktaDefaults.ApiAuthenticationType)`.